DQM/SiPixelPhase1Common/python/SpecificationBuilder_cfi.py

```diff
@@ -0,0 +1,116 @@
+import FWCore.ParameterSet.Config as cms
+
+# this is a pure Python module to build the deeply nested PSets that describe a 
+# SummationSpecification.
+# The C++ code assumes the form is fully correct, so you should always use this, 
+# which outputs a valid form.
+
+# these need to stay in sync with the C++ enums.  TODO: Can we use Python3 enum or sth.?
```

Comment:
This is a option to consider. The related problem there is that afaik there is no really nice way to do the string-enum conversion in C++.